README.md

```diff
@@ -40,10 +41,16 @@ brief description of what it does.
 * [cubeseer](https://github.com/justinj/cubeseer) - Diagram generator, but contains a cube representation
 * [RaMSDeL](https://github.com/justinj/ramsdel) - Random Move Scrambler Definition Language
 
-#C++
-
+C++
+===
```

Comment:
I don't really care too much, I already changed it to all use the `===` way, but I don't mind changing it back. C++ needed the explicit link to get the linking from the top to work.